fe/fe-core/src/main/java/org/apache/doris/load/routineload/KafkaRoutineLoadJob.java

```diff
@@ -17,6 +17,7 @@
 
 package org.apache.doris.load.routineload;
 
+import java.util.HashMap;
```

Comment:
Pay attention to the ordering of imports

Reply:
ok, I'll adjust it later